ogr/services/mock/core.py

```diff
@@ -0,0 +1,82 @@
+import os
+import inspect
+import yaml
+
+def write_output(*args):
+    print(*args)
+
+class CallLogger(object):
+    def __init__(self, namespace, class_obj):
+        self.namespace = namespace
+        self.class_obj = class_obj
+
+    def logcall(self, method_name, args, kwargs):
+        output = "namespace:{} class: {} call: {}".format(self.namespace, self.class_obj.__name__, method_name)
+        output += ("(")
+        if args or kwargs:
+
+            if len(args) == 1:
+                output += str(args)[1:-2]
+            elif len(args) > 1:
+                output += str(args)[1:-1]
+            if kwargs:
+                if args:
+                    output += ", "
+                output += "**{}".format(kwargs)
+        output += (")")
+        write_output(output)
+        return output
+
+
+class MockClass(object):
+    configuration_file = "mock.yaml"
```

Comment:
I would prefer not to use any files at all.

(The main use case, for now, is `--dry-run` in packit -- print only the called methods.)

Reply:
yes, this is optional. and main purpose is to store return values for testing, for dry-run it is not useful.

Reply:
I would place testing stuff to `./tests/` and not mess with the regular code.